tests/integration_tests/functional/test_shut_down.py

```diff
@@ -14,7 +14,6 @@ def test_reboot(test_microvm_with_api):
     Test reboot from guest.
     """
     vm = test_microvm_with_api
-    vm.jailer.daemonize = False
```

Comment:
While developing this commit I found this was not needed by the test. I included it here, but I could also split it out as a separate commit if that helps.